lib/plan-builder-generated.js

```diff
@@ -1155,10 +1154,9 @@ periodRangeQuery(...args) {
 point(...args) {
     const namer = bldrbase.getNamer(args, 'latitude');
     const paramdefs = [['latitude', [types.XsDouble, PlanColumn, PlanParam], true, false], ['longitude', [types.XsDouble, PlanColumn, PlanParam], true, false]];
-    // Changing minArity to 1 manually since cts.point now accepts string input (MarkLogic 11.1.0 or higher needed).
     const checkedArgs = (namer !== null) ?
         bldrbase.makeNamedArgs(namer, 'cts.point', 2, new Set(['latitude', 'longitude']), paramdefs, args) :
-        bldrbase.makePositionalArgs('cts.point', 1, false, paramdefs, args);
+        bldrbase.makePositionalArgs('cts.point', 2, false, paramdefs, args);
```

Comment:
This looks like an incorrect change, as 11.1.0 added the ability for cts.point to accept one arg.